### PR TITLE
[844] Reformat the code for Quartile modules

### PR DESCRIPTION
**Pull request by YogeshMahera-SerpentCS**
- Task [[844](https://www.quartile.co/web#id=844&action=771&model=project.task&view_type=form&menu_id=505)]
- Closed PR #269

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ env:
     # Use the above line to install dependencies that are required for website repos:
     # * SASS & Bootstrap-SASS
     # * Compass
-    - EXCLUDE=""
+    - EXCLUDE="odoo_marketplace,connector_odoo_sst,restful,component_event,access_apps,base_technical_features,drag-and-drop,mail_bcc,web_responsive,access_restricted,component,ir_rule_protected,queue_job,website_no_crawler,access_settings_menu,connector,l10n_jp_country_state,web_debranding,website_sale_search_clear,base_country_state_translatable,dbfilter_from_header,l10n_jp_partner_title_qweb,web_environment_ribbon"
```
Comment (TimLai125) [inline .travis.yml]: @YogeshMahera-SerpentCS Please minimize the `Exclude` and put the failed test method as reference if it is inside the `Exclude`.

**Review by TimLai125 — CHANGES_REQUESTED** [review]

**Comment by YogeshMahera-SerpentCS** [comment]
Hi @TimLai125 updated changes can you review it?

File: l10n_jp_address_layout/__manifest__.py [hunk 2]
```diff
@@ -1,23 +1,16 @@
-# -*- coding: utf-8 -*-
```
Comment (TimLai125) [inline l10n_jp_address_layout/__manifest__.py]: This module is an OCA module, i.e. https://github.com/OCA/l10n-japan/tree/11.0/l10n_jp_address_layout
Please exclude it from pre-commit and revert to the original version.

File: purchase_ext_sst/models/purchase_order_line_hook_onchange_product_id.py [hunk 3]
```diff
@@ -1,45 +1,55 @@
...
 from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
 from odoo.addons.purchase.models.purchase import PurchaseOrderLine
 
+
 # Monkey Patching
 # Overwrite the original onchange_product_id
-# i.e. https://github.com/odoo/odoo/blob/11.0/addons/purchase/models/purchase.py#L781-L811
-@api.onchange('product_id')
+# i.e. https://github.com/odoo/odoo/blob/11.0
```
Comment (TimLai125) [inline purchase_ext_sst/models/purchase_order_line_hook_onchange_product_id.py]: Please use `# noqa` for this line, we do not want to split links into two lines.

File: stock_quant_sale_order/wizard/stock_quant_sale_order.py [hunk 4]
```diff
@@ -28,77 +19,89 @@ def default_get(self, fields):
...
         sale_order.onchange_partner_id()
         sale_order.team_id = self.team_id.id
         sale_order.user_id = self.env.uid
 
         # A list to store the values of sale.order.line's required fieds
-        order_lines_value_list = []
+        # order_lines_value_list = []
         # A dict to store the tax_id of each product
-        product_tax_id = {}
+        # product_tax_id = {}
```
Comment (TimLai125) [inline stock_quant_sale_order/wizard/stock_quant_sale_order.py]: Please delete these lines if the variables are not being used

File: website_sale_stock_virtual_availability/controllers/main.py [hunk 5]
```diff
@@ -1,22 +1,25 @@
...
 from odoo.tools.pycompat import izip
 
+from odoo.addons.website_sale_stock.controllers.main import WebsiteSale
+
 
 class WebsiteSale(WebsiteSale):
 
     # Add website_sale_available_qty to the return value of
     # get_attribute_value_ids that the value will be retrieved by js
-    # i.e. https://github.com/odoo/odoo/blob/11.0/addons/website_sale_stock/controllers/main.py#L11-L27
```
Comment (TimLai125) [inline website_sale_stock_virtual_availability/controllers/main.py:13]: Please apply # noqa as well

**Review by TimLai125 — CHANGES_REQUESTED** [review]
Github does not recognize some of the python files, @YogeshMahera-SerpentCS  any idea?
Please remove the "description" in the `__manifest__.py` in case the description is already in the README.rst file.

**Comment by YogeshMahera-SerpentCS** [comment]
> Github does not recognize some of the python files, @YogeshMahera-SerpentCS any idea?
Please remove the "description" in the __manifest__.py in case the description is already in the README.rst file.

Hi @TimLai125 I have no idea but maybe there is too many files to check in this PR so maybe that's why. also amended changes.

**Pull request merged**